Library/Formula/iojs.rb

```diff
@@ -0,0 +1,39 @@
+class Iojs < Formula
+  homepage "https://iojs.org/"
+  url "https://iojs.org/dist/v1.0.4/iojs-v1.0.4.tar.xz"
+  version "1.0.4"
```

Comment:
This might be redundant from the ` url `. If it is, it should show up under ` brew audit `.